README.md

```diff
@@ -186,9 +186,15 @@ import AirtimeMedia
 
 // ... create a channel
 
-let localStream = channel.startLocalStream()
+let localStream = channel.createLocalStream()
 guard let localStream = localStream else { fatalError() } // Handle error case
 
+// Start the local stream
+localStream.start()
+
+// Mute/unmute the audio
+localStream.setAudioMuted(true)
```

Comment:
Maybe we should have a small section on muting/unmuting. It would be useful to describe a bit how muting/unmuting is async but muting should always succeed whereas unmuting depends on the server response and token permissions

Reply:
Added in latest.